src/auto-evo/steps/IncreaseBiodiversity.cs

```diff
@@ -20,8 +20,12 @@ public class IncreaseBiodiversity : IRunStep
         private bool tryCurrentPatch = true;
         private bool createdASpecies;
 
-        public IncreaseBiodiversity(AutoEvoConfiguration configuration, PatchMap map, Patch patch, Random random)
+        private WorldGenerationSettings settings;
+
+        public IncreaseBiodiversity(AutoEvoConfiguration configuration,
+            WorldGenerationSettings settings, PatchMap map, Patch patch, Random random)
```

Comment:
I'd suggest naming `settings` `worldSettings` here so that there aren't two very similarly named things, configuration and settings.